tests/system/Events/EventsTest.php

```diff
@@ -55,21 +55,21 @@ public function testInitialize()
 
         // it should start out empty
         MockEvents::setFiles([]);
-        $this->assertEmpty($this->manager->getFiles());
+        $this->assertEmpty(MockEvents::getFiles());
```

Comment:
Please revert.

Testing mocks is a bad practice. It is inherently wrong. 
Because mocks are not real product code.
Mocks are tools for testing. The test target should be the real product code, not the mock.

However, in this test case, there is no other way but to test the mock because the design of Event is not good.

Still, the code to check the mock directly is uncomfortable. 
The `$this->manager` represents the test target, which is better than checking MockEvents directly.

Reply:
Ok, I understand the logic of the testing mock and sorry for this terrible change.
I have been revert this test case change.

Reply:
Thanks for the fix.